libr/core/cmd_open.c

```diff
@@ -328,9 +328,17 @@ static int cmd_open(void *data, const char *input) {
 	case '\0':
 	case '*':
 	case 'j':
-		r_core_file_list (core, (int)(*input));
+		if ('?' == input[1]) {
+			eprintf ("Usage: oj\n");
```

Comment:
not completelly correct. because this is also the case for '*' and '0'... also help messages are suposed to be rendered to stdout via the r_core_help api instead of eprintf which is suposed to be used only for error messages